### ISSUE TITLE
Topic 7 Color Contrast failure still exists

### ISSUE THREAD

**Issue by vprentice** [Section508Coordinators/DHSA-TT-200-A#71]
## Description
Color Contrast failure- Universal

## Location
https://training.section508testing.net/mod/book/view.php?id=1238&chapterid=2824

**Comment by drewnielson** [comment]
Fixed by Kat; revised stylesheet for books as well.
